doc/nrf/ug_bootloader_external_flash.rst

```diff
@@ -0,0 +1,67 @@
+.. _ug_bootloader_external_flash:
+
+Using external flash memory partitions
+######################################
+
+It is possible to use the external flash memory as the storage partition for the secondary slot.
+This requires a driver for the external flash memory that supports the following:
+
+* Single-byte read and write.
+* Writing data from the internal flash memory to the external flash memory.
+
+See :ref:`pm_external_flash` for general information about how to set up partitions in the external flash memory in the Partition Manager.
+
+.. note::
+
+   External flash memory partitions for image slots are currently only supported by MCUboot.
+
+To configure a bootloader to use the external flash memory for the secondary slot, edit the configuration file of the bootloader child image to specify an external flash region.
+For MCUboot, this configuration is the file :file:`ncs/bootloader/mcuboot/boot/zephyr/pm.yml`, and should have the region defined under ``mcuboot_secondary``:
+
+.. code-block:: yaml
+
+   mcuboot_secondary:
+       region: external_flash
+       size: CONFIG_PM_EXTERNAL_FLASH_SIZE
+
+The following example shows how to configure an application for the nRF52840 DK.
+The nRF52840 DK comes with the external flash memory that can be used for the secondary slot and that can be accessed using the QSPI NOR flash memory driver.
+
+1. Add the following configuration options to the :file:`ncs/bootloader/mcuboot/boot/zephyr/prj.conf` file:
+
+   .. code-block:: console
+
+      CONFIG_NORDIC_QSPI_NOR=y
+      CONFIG_NORDIC_QSPI_NOR_FLASH_LAYOUT_PAGE_SIZE=4096
+      CONFIG_NORDIC_QSPI_NOR_FLASH_ALLOW_STACK_USAGE_FOR_DATA_IN_FLASH=y
```

Comment:
That option seems to not exist, so adding it to the prj.conf will end with compiling error. I believe it should be located in the zephyr/drivers/flash/Kconfig.nordic_qspi_nor, as other options named like CONFIG_NORDIC_QSPI_NOR but there isn't.

Reply:
This option was removed, it was needed for hack the driver for MCUboot usage which is no longer needed.